packages/dso-toolkit/src/components/tabs/tabs.models.ts

```diff
@@ -1,10 +1,17 @@
 export interface Tabs<TemplateFnReturnType> {
-  items: TabsItem<TemplateFnReturnType>[];
+  items: TabsItem[];
+  content: TemplateFnReturnType | string;
 }
 
-export interface TabsItem<TemplateFnReturnType> {
-  label: string;
-  id: string;
+export interface TabsItem {
+  label?: string;
```

Comment:
Functioneel is er gespect dat er altijd een label is.

Reply:
Gedaan.